.github/workflows/publish-image-with-version.yml

```diff
@@ -0,0 +1,107 @@
+name: Create and publish Switchover Agent Docker image with version
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Does this mean we must pass in a tag to trigger the image build?

Reply:
The create release action would create a tag when creating a release